lib/tasks/evm_application.rb

```diff
@@ -110,4 +110,13 @@ def self.update_stop
     File.write(tempfile, " ")
     stop
   end
+
+  def self.set_region_file(region_file, new_region)
+    old_region = region_file.exist? ? region_file.read.to_i : nil
+
+    return if new_region == old_region
+
+    _log.info("Changing REGION file from [#{old_region}] to [#{new_region}]. Restart to use the new region.")
+    region_file.write(new_region)
```

Comment:
I don't think we *need* to, but I'm sure there are things that still rely on it.

When it came down to it, that was all `join_region.rb` was doing, so I figured we should keep it.